modules/core/04-channel/keeper/upgrade.go

```diff
@@ -319,7 +319,7 @@ func (k Keeper) ChanUpgradeOpen(
 	portID,
 	channelID string,
 	counterpartyChannelState types.State,
-	proofChannel []byte,
+	proofCounterpartyChannel []byte,
```

Comment:
I think I'd prefer to keep this as `proofChannel` as we are always verify proofs of a counterparty (its implicit)

Reply:
I do think `proofChannel` caused my initial mis-step on this. Other handles also prefer explicit. Change later all together if we decide so?

Reply:
think there's some areas where different approaches have been taken (e.g also how we do connection look-up) deviating from the old handlers, definitely would prefer to hammer those differences out and be consistent. Opened https://github.com/cosmos/ibc-go/issues/4053 to keep track of it and not forget